spring-cloud-gcp-vision/src/test/java/com/google/cloud/spring/vision/it/DocumentOcrTemplateIntegrationTests.java

```diff
@@ -42,16 +43,18 @@
 class DocumentOcrTemplateIntegrationTests {
 
   @Autowired private DocumentOcrTemplate documentOcrTemplate;
+  @Value("${vision-integration-test-bucket}")
+  private String testBucketName;
 
   @Test
   void testDocumentOcrTemplate()
       throws ExecutionException, InterruptedException, InvalidProtocolBufferException,
           TimeoutException {
 
     GoogleStorageLocation document =
-        GoogleStorageLocation.forFile("vision-integration-test-bucket", "test.pdf");
+        GoogleStorageLocation.forFile(testBucketName, "test.pdf");
     GoogleStorageLocation outputLocationPrefix =
-        GoogleStorageLocation.forFile("vision-integration-test-bucket", "it_output/test-");
+        GoogleStorageLocation.forFile(testBucketName, "it_output/test-");
```

Comment:
> exposed test resource in DocumentOcrTemplateIntegrationTests.java so it can be set explicitly for native test. (storage bucket name are global unique, thus cannot reuse same name from regular ci.

Rather than requiring parallel tests using distinct buckets (which isn't going to scale to the number of parallel invocations we want to eventually have running), could we just add a test-instance-specific ID to the output path, and always use the same bucket?

Reply:
Makes sense. Modified the output path to contain a random id suffix. Let me know if this is what you had in mind?